contracts/SortitionPool.sol

```diff
@@ -56,16 +56,27 @@ contract SortitionPool is SortitionTree, Rewards, Ownable, IReceiveApproval {
     Rewards.addRewards(uint96(amount), uint32(root.sumWeight()));
   }
 
+  /// @notice Withdraws all available rewards for the given operator to the
+  ///         given beneficiary.
+  /// @dev Can be called only be the owner. Does not validate if the provided
+  ///      beneficiary is associated with the provided operator - this needs to
+  ///      be done by the owner calling this function.
+  /// @return The amount of rewards withdrawn in this call.
   function withdrawRewards(address operator, address beneficiary)
     public
     onlyOwner
+    returns (uint96)
   {
     uint32 id = getOperatorID(operator);
     Rewards.updateOperatorRewards(id, uint32(getPoolWeight(operator)));
     uint96 earned = Rewards.withdrawOperatorRewards(id);
     rewardToken.transfer(beneficiary, uint256(earned));
+    return earned;
```

Comment:
Not going to block this PR for it, but I just merged #162 that uses this function in tests.

Now that we have a return value, we should assert that the return value of `withdrawRewards` is correct!